lib/lz4.c

```diff
@@ -188,17 +191,38 @@ static void LZ4_write16(void* memPtr, U16 value) { ((unalign*)memPtr)->u16 = val
 
 static U16 LZ4_read16(const void* memPtr)
 {
+#ifdef __TI_COMPILER_VERSION__
+    /* Suppess warning: #551-D variable "ctx" is used before its value is set*/
```

Comment:
Maybe the action here would be to remove the warning.
Since we never met it during our tests, maybe some information on the test conditions would be useful.